srcSpecs/demo-spec.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape');var file = require('../src/demo.js');test('sum function', function(t) { t.plan(2); 
```

Comment:
It was added to the PR, but even if you didn't have it, the file is generated when you run the command ```node speck.js tape srcSpecs/ src/demo.js```